go.mod

```diff
@@ -11,19 +11,19 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/evanphx/json-patch/v5 v5.6.0
 	github.com/go-git/go-git/v5 v5.1.0
-	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/logr v1.2.4
 	github.com/google/go-cmp v0.5.9
-	github.com/prometheus/client_golang v1.14.0
+	github.com/prometheus/client_golang v1.15.1
 	golang.org/x/crypto v0.1.0
-	golang.org/x/tools v0.3.0
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
```

Comment:
@stephanhorsthemke,

Thank you a lot for this PR. Could you check the errors in the CI test?
We need to fix those before running the others.